deploy/catalog/README.md

```diff
@@ -4,31 +4,77 @@ The Helm Chart deploys Service Catalog into an existing Kubernetes cluster.
 
 ## Prerequisites
 
-  - [Helm](https://github.com/kubernetes/helm) must be installed in the cluster.
-  - Images must be built from source and pushed to a registry accessible to
-    the cluster. Example (from this directory):
-
-    ```
-    $ export REGISTRY=hub.docker.com/<username>
-    $ make -C ../.. push
-    ```
-
-    __Note:__ If deploying locally (e.g. in Minikube), it can be more efficient
-    to use a _local_ Docker registry.
+- [Helm](https://github.com/kubernetes/helm) must be installed in the cluster
+- Images must be built from source and pushed to a registry accessible to the
+  cluster if deploying to a non-local cluster; see the [dev guide](../../docs/DEVGUIDE.md) for instructions
 
 ## Usage
 
 Supported template parameters (values):
 
-  - `registry`  (required): Container registry with Service Catalog images.
-  - `version`   (optional): Version of Service Catalog (container images) to deploy.
-  - `namespace` (optional): A Kubernetes namespace to use for Service Catalog deployment.
-  - `storageType` (optional): The type of storage for the API server to use.
-  - `verbosity` (optional): The verbosity of logs.
-  - `debug` (optional): create a load balancer for the api server service
+### Deployment Knobs
+
+- `registry`: Container registry with Service Catalog images; optional; defaults
+  to unset
+- `namespace`: A Kubernetes namespace to use for deployment of this
+  chart into k8s; optional; defaults to `default`
+- `version`: Version of Service Catalog (container images) to deploy; optional;
+  defaults to `latest`
+- `apiServerVersion`: The version of the API server image to deploy; optional;
+  defaults to unset; overrides `version` for the API server if set
+- `controllerManagerVersion`: The version of the controller-manager image to
+  deploy; optional; defaults to unset; overrides `version` for the
+  controller-manager if set
+- `etcdImage`: The name of the etcd image to use; optional; defaults to
+  `quay.io/coreos/etcd`
+- `etcdVersion`: The version of the etcd image to deploy; optional; defaults to 
+  unset; overrides `version` for etcd if set
+
+### Catalog knobs
+
+- `verbosity`: The verbosity of logs; optional; defaults to `10`
+- `debug`: Whether to create a load balancer for the apiserver and
+  controller-manager services; optional; defaults to `false`
+- `imagePullPolicy`: The image pull policy to use for all pods in this chart;
+  optional; defaults to `Always`
```

Comment:
This default is fine for now while we hack. The default should change to `IfNotPresent` in the future.